jobserver/models/core.py

```diff
@@ -525,7 +525,7 @@ class Meta:
         ]
 
     def __str__(self):
-        return f"{self.org.name} | {self.name}"
+        return f"{self.org.name} | {self.title}"
```

Comment:
When looking through the list of projects to set up a user I realised we weren't displaying the project number, which is included when we use `title`.  I figured it would be a little easier to find the projects with that visible!